integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeExceptionHandler.java

```diff
@@ -103,39 +124,41 @@ public void handleRemoteApiException(RemoteApiException remoteException,
       IntegrationInstance instance, String integrationUser, String stream) {
     int code = remoteException.getCode();
     Status status = Status.fromStatusCode(code);
+    String message = logMessage.getMessage(UNABLE_POST_STREAM, stream, String.valueOf(code));
 
-    LOGGER.error(
-        String.format("The Integration Bridge was unable to post to stream %s due to error code %d",
-            stream, code), remoteException);
+    LOGGER.error(message,remoteException);
 
     if (forbiddenError(code)) {
       updateStreams(instance, integrationUser, stream);
     } else if (Status.BAD_REQUEST.equals(status)) {
-      String logMessage =
-          String.format("Invalid message posted to stream %s.\nInstance: %s", stream,
-              instance.getInstanceId());
-      LOGGER.warn(logMessage, remoteException);
+      LOGGER.warn(logMessage.getMessage(INVALID_MESSAGE, stream, instance.getInstanceId()),
+          remoteException);
     }
   }
 
   public void handleUnexpectedException(Exception e) {
-    LOGGER.error("Fail to post message", e);
+    LOGGER.error(logMessage.getMessage(FAIL_POST_MESSAGE), e);
```

Comment:
Generic message with solution. The caller also rethrows this exception.

We shouldn't log the message and rethrow the exception. Probably, this exception will be logged by other layer.